vNext/test/TesterInternal/TesterInternal.csproj

```diff
@@ -50,8 +50,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\..\test\TesterInternal\**\*.cs" Exclude="..\..\..\test\TesterInternal\obj\**\*.cs;..\..\..\test\TesterInternal\bin\**\*.cs;..\..\..\test\TesterInternal\Properties\*.cs;..\..\..\test\TesterInternal\CounterControlProgTests.cs;..\..\..\test\TesterInternal\GeoClusterTests\*.cs;..\..\..\test\TesterInternal\HostedTestClusterBase.cs;..\..\..\test\TesterInternal\OrleansHostProgTests.cs;..\..\..\test\TesterInternal\General\BootstrapProviderTests.cs;
-    ..\..\..\test\TesterInternal\StorageTests\PersistenceGrainTests.cs;..\..\..\test\TesterInternal\StreamingTests\StreamPubSubReliabilityTests.cs" />
+    <Compile Include="..\..\..\test\TesterInternal\**\*.cs" Exclude="..\..\..\test\TesterInternal\obj\**\*.cs;..\..\..\test\TesterInternal\bin\**\*.cs;..\..\..\test\TesterInternal\Properties\*.cs;..\..\..\test\TesterInternal\CounterControlProgTests.cs;..\..\..\test\TesterInternal\GeoClusterTests\*.cs;..\..\..\test\TesterInternal\HostedTestClusterBase.cs;..\..\..\test\TesterInternal\OrleansHostProgTests.cs;..\..\..\test\TesterInternal\General\BootstrapProviderTests.cs;&#xD;&#xA;    ..\..\..\test\TesterInternal\StorageTests\PersistenceGrainTests.cs;..\..\..\test\TesterInternal\StreamingTests\StreamPubSubReliabilityTests.cs" />
```

Comment:
I suppose the `&#xD;&#xA;` is an encoding of the newline, but maybe we should just remove all whitespace